test/parse-outputs/test_parse_outputs.py

```diff
@@ -1,13 +1,15 @@
 import filecmp
 from pathlib import Path
 
+import pytest
+
 from spras import runner
 
-INDIR = "test/parse-outputs/input/"
-OUTDIR = "test/parse-outputs/output/"
-EXPDIR = "test/parse-outputs/expected/"
-OI2_EDGE_CASES_INDIR = 'test/parse-outputs/input/omicsintegrator-edge-cases/'
-DUPLICATE_EDGES_DIR = 'test/parse-outputs/input/duplicate-edges/'
+INDIR = Path("test", "parse-outputs", "input")
```

Comment:
I saw a lot of updates to the filepaths in many of the PRs, was there a reason for this?

Reply:
Is there a reason for doing this?